notes/2019-04-05-kick-off-week/README.md

```diff
@@ -117,7 +129,7 @@ We wrapped up this session with a couple standing questions to be addressed the
 ## Roadmap & Continuity
 
 In 2016, DWeb Summit involved mostly builders of technologies. 
-In 2018, participants represented multiple disciplines and efforts were made to broaden gender diversity.
+In 2018, participants represented multiple disciplines and efforts were made to broaden gender, ethnic and geographic diversity.
 In organizing the third event, our first Camp, we hope to more deeply explore who is represented and create more meaningful representations by curating for truly inclusive spaces.
```

Comment:
instead of "by curating for truly inclusive spaces" how about "by thoughtful curation, supportive structures like support circles, and encouraging multi-disciplinary projects that span many groups."

Reply:
```suggestion
In organizing the third event, our first Camp, we hope to more deeply explore who is represented and create more meaningful representations by thoughtful curation, supportive structures like support circles, and encouraging multi-disciplinary projects that span many groups.
```